packages/axelar-wasm-std/src/verification.rs

```diff
@@ -0,0 +1,12 @@
+use cosmwasm_schema::cw_serde;
+
+#[cw_serde]
+#[derive(Copy)]
+pub enum VerificationStatus {
+    SucceededOnChain,
+    FailedOnChain,
+    NotFound,
+    FailedToVerify, // verification process failed, e.g. no consensus reached
+    InProgress,     // verification in progress
+    NotVerified,    // not yet verified, e.g. not in a poll
```

Comment:
```suggestion
    NotInAPoll
```
or 
```suggestion
    None
```

I just feel that `NotVerified` is a little confusing, because some of the other statuses also seem like they could be considered "not verified" (like `FailedToVerify` or `InProgress`)